README.md

```diff
@@ -1 +1,43 @@
-# leapp-report-analyzer
\ No newline at end of file
+# Leapp Report Analyzer
+
+Leapp Report Analyzer is a command-line tool for analyzing multiple Leapp reports and grouping systems by severity and entries.
+The tool processes JSON files containing Leapp reports and generates a JSON file and an HTML table displaying the results.
+
+By analyzing Leapp reports and providing organized output, the Leapp Report Analyzer simplifies the process of identifying and addressing potential issues in your systems.
+
+### Requirements
+
+Script works on both Python2 and Python3, tested with:
+
+- Python 2.7
+- Python 3.3
+
+### Usage
+
+You can use the Leapp Report Analyzer with either individual Leapp report JSON files or a directory containing multiple JSON files. The following arguments are required:
+
+- `-f / --input-files`: One or more Leapp report JSON files to be analyzed (e.g., `-f report1.json report2.json`).
+- `-d / --input-directory`: A directory containing Leapp report JSON files to be analyzed (e.g., `-d reports_directory`).
```

Comment:
Both, or only one of them is required? based on code they are exclusive, so -f or -d